js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
 times.days=Tage
 user=Benutzer
 username=Benutzername
+loginScreenCustomization=Anmeldeformular Anpassungen
+registrationAllowed=Benutzerregistrierung
+forgotPassword=Passwort vergessen
+forgotPasswordHelpText=Zeigt einen Link an, auf den der Benutzer klicken kann, wenn er seine Anmeldedaten vergessen hat.
+emailSettings=E-Mail Einstellungen
+duplicateEmailsAllowed=E-Mail Duplikate erlauben
+duplicateEmailsAllowedHelp=Zulassen, dass mehrere Benutzer dieselbe E-Mail-Adresse haben können. Wenn Sie diese Einstellung ändern, wird auch der Cache des Benutzers gelöscht. Es wird empfohlen, die E-Mail-Beschränkungen bestehender Benutzer in der Datenbank manuell zu aktualisieren, nachdem die Unterstützung für doppelte E-Mail-Adressen deaktiviert wurde.
+userInfoSettings=Benutzer Einstellungen
+connectionAndAuthentication=Verbindung & Authentifizierung
+encryption=Verschlüsselung
+enableStartTLS=StartTLS aktivieren
+testConnectionHint.withoutEmail=Um die Verbindung zu testen, müssen Sie zunächst eine E-Mail-Adresse für den aktuellen Benutzer ({{userName}}) konfigurieren.
+testConnectionHint.withoutEmailAction=E-Mail Adresse konfigurieren
+testConnection=Verbindung testen
+smtpPortPlaceholder=SMTP Port (Standardwert 25)
+envelopeFromHelp=Eine E-Mail-Adresse, die für Bounces (nicht zustellbaren E-Mails) verwendet wird (optional).
+resetPasswordAllowed=Passwort vergessen
+disabled=Nicht aktiv
+themes=Thema
+selectATheme=Thema auswählen
```

Comment:
Entweder Theme oder Design. Thema would be `topic` in English

Reply:
Then I rather stick with "Theme" here. I'll adapt the translations.